include/xsimd/arch/xsimd_neon.hpp

```diff
@@ -452,59 +452,112 @@ namespace xsimd
          * load *
          ********/
 
+        // It is not possible to use a call to A::alignment() here, so use an
+        // immediate instead.
+#if defined(__clang__) || __GNUC__
+#define xsimd_aligned_load(inst, type, expr) inst((type)__builtin_assume_aligned(expr, 16))
+#elif defined(_MSC_VER)
+#define xsimd_aligned_load(inst, type, expr) inst##_ex((type)expr, 16)
```

Comment:
Don't know these `_ex`functions, should the alignment be 128 (in bits, not bytes)?
https://github.com/ermig1979/Simd/blob/master/src/Simd/SimdLoad.h#L677-L690

Reply:
Indeed, according to the winsdk file, for instance:
- https://github.com/tpn/winsdk-10/blob/master/Include/10.0.14393.0/km/crt/arm_neon.h#L1043 where the alignment is passed to `_NEON_ALIGN64_128`
- https://github.com/tpn/winsdk-10/blob/master/Include/10.0.14393.0/km/crt/arm_neon.h#L371 for the definition of `_NEON_ALIGN64_128`